doc/support.rst

```diff
@@ -2,96 +2,71 @@
 
-In both cases please use a descriptive question in the title field (e.g.
-no "Please help with scikit-learn!" as this is not a question) and put
-details on what you tried to achieve, what were the expected results and
-what you observed instead in the details field.
+- **General Machine Learning Queries**: For broader machine learning discussions, visit `Stack Exchange <https://stats.stackexchange.com/>`_.
 
-Code and data snippets are welcome. Minimalistic (up to ~20 lines long)
-reproduction script very helpful.
+When posting questions:
 
-Please describe the nature of your data and how you preprocessed it:
-what is the number of samples, what is the number and type of features
-(i.d. categorical or numerical) and for supervised learning tasks,
-what target are your trying to predict: binary, multiclass (1 out of
-``n_classes``) or multilabel (``k`` out of ``n_classes``) classification
-or continuous variable regression.
+- Please use a descriptive question in the title field (e.g. no "Please help with scikit-learn!" as this is not a question) 
 
-User questions should **not be asked on the bug tracker**, as it crowds
-the list of issues and makes the development of the project harder.
+- Provide detailed context, expected results, and actual observations.
 
-.. _bug_tracker:
+- Include code and data snippets (preferably minimalistic scripts, up to ~20 lines).
 
-Bug tracker
-===========
+- Describe your data and preprocessing steps, including sample size, feature types (categorical or numerical), and the target for supervised learning tasks (classification type or regression).
 
-If you think you've encountered a bug, please report it to the issue tracker:
+**Note**: Avoid asking user questions on the bug tracker to keep the focus on development.
 
-https://github.com/scikit-learn/scikit-learn/issues
+.. _bug_tracker:
+
+Bug Tracker
+===========
 
-Don't forget to include:
+Encountered a bug? Report it on our `issue tracker
+<https://github.com/scikit-learn/scikit-learn/issues>`_
 
-- steps (or better script) to reproduce,
+Include in your report:
 
-- expected outcome,
+- Steps or scripts to reproduce the bug.
 
-- observed outcome or Python (or gdb) tracebacks
+- Expected and observed outcomes.
 
-To help developers fix your bug faster, please link to a https://gist.github.com
-holding a standalone minimalistic python script that reproduces your bug and
-optionally a minimalistic subsample of your dataset (for instance, exported
-as CSV files using ``numpy.savetxt``).
+- Python or gdb tracebacks, if applicable.
 
-Note: Gists are Git cloneable repositories and thus you can use Git to
-push datafiles to them.
+Enhance your report by crafting a `minimal reproducer for scikit-learn <https://scikit-learn.org/stable/developers/minimal_reproducer.html>`_, which includes a standalone script and a sample of your dataset (use ``numpy.savetxt`` for exporting CSV files).
```

Comment:
```suggestion
- The ideal bug report contains a :ref:`short reproducible code snippet
  <minimal_reproducer>`, this way anyone can try to reproduce the bug easily
  (see :ref:`filing_bugs` for more details). If your snippet is longer than
  around 50 lines, please link to a `gist <https://gist.github.com>`_ or a
  github repo.
```

Reply:
":ref:`filing_bugs`", I tried your suggestion but the filing_bugs page is not found. Could you guide me to where this page exists please @ArturoAmorQ